components/list/test/list.vdiff.js

```diff
@@ -180,9 +180,10 @@ function createButtonItem({ disabled = false, template = 'Item 1' } = {}) {
 	`;
 }
 
-function createItems({ paddingType, width, withColors = false } = {}) {
+function createItems({ lineBreak, paddingType, width, withColors = false } = {}) {
 	return html`
 		${createItem({ color: withColors ? '#006fbf' : undefined, paddingType, template: 'Item 1', width })}
+		${lineBreak ? html`<div class="d2l-list-item-break"></div>` : nothing}
```

Comment:
Alternatively, we could provide a `d2l-list-item-break` component, but it seems like overkill. I'm not opposed to it.

Reply:
So it's a line break but only in tile view? Maybe `d2l-list-item-break-tile-only`? Having a component would allow us to set `aria-role="separator"` and then it'd behave more like a `<br>`.